ci-operator/step-registry/openshift/e2e/test-iterations/openshift-e2e-test-iterations-commands.sh

```
@@ -0,0 +1,68 @@
+external) export TEST_PROVIDER='{"type":"external"}';;
+*) echo >&2 "Unsupported cluster type '${CLUSTER_TYPE}'"; exit 1;;
+esac
+
+mkdir -p "${HOME}"
+mkdir -p /tmp/output
+cd /tmp/output
+
+TEST_ARGS="${TEST_ARGS:-} ${SHARD_ARGS:-}"
+
+# Run the test suite multiple times
+for i in $(seq 1 ${TEST_ITERATIONS}); do
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)